### ISSUE TITLE
whitelisted_ransackable_associations set to 'true' if previously unset

### ISSUE THREAD

**Issue by mvz** [spree-contrib/spree_i18n#629]
In [app/models/spree_i18n/translatable.rb](https://github.com/spree-contrib/spree_i18n/blob/1a2791cedd1d06f45e5976737942049aea3dadd4/app/models/spree_i18n/translatable.rb), the following line only works if `whitelisted_ransackable_associations` was previously an array:

```
klass.whitelisted_ransackable_associations |= ['translations']
```

For some models (e.g., `Spree::Promotion`), this value is not set by plain Spree at all, causing it to be set to `true`. This leads to errors:

```
NoMethodError: undefined method `include?' for true:TrueClass
```

**Comment by mvz** [comment]
All this is on branch `3-0-stable`.

**Comment by mvz** [comment]
Fixed with #634.
